packages/tools/client-debugger/client-debugger-view/src/components/FluidDataView.tsx

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { HasContainerId, VisualNode, VisualNodeKind } from "@fluid-tools/client-debugger";
+import { FluidHandleView } from "./FluidHandleView";
+import { TreeView } from "./TreeView";
+import { FluidTreeView } from "./FluidTreeView";
+import { ValueView } from "./ValueView";
+import { FluidValueView } from "./FluidValueView";
+import { UnknownFluidObjectView } from "./UnknownFluidObjectView";
+import { UnknownDataView } from "./UnknownDataView";
+import { Waiting } from "./Waiting";
+import { waitingLabels } from "./WaitingLabels";
+
+/**
+ * {@link FluidDataView} input props.
+ */
+export interface FluidDataViewProps extends HasContainerId {
+	node: VisualNode;
+}
+
+/**
+ * Displays visual summary trees for DDS_s within the container.
+ */
+export function FluidDataView(props: FluidDataViewProps): React.ReactElement {
```

Comment:
We should probably call this something like `TreeView`, since this isn't strictly for Fluid-related visual trees.